kronos-java/src/main/java/com/lyft/kronos/internal/ntp/SntpClient.java

```diff
@@ -73,6 +72,8 @@ public SntpClient(Clock deviceClock, DnsResolver dnsResolver, DatagramFactory da
         this.datagramFactory = datagramFactory;
     }
 
+    private final NtpPackets ntpPackets = new NtpPackets.Impl();
```

Comment:
I’ll pass the instance via the constructor in next PRs. The goal of this one was to show that the current test suite passes using the new protocol implementation.